### PR TITLE
Fixes #733 (missing feedback-notification-emails) by rebuilding the feature

### LINKED ISSUE #733: No feedback-received acknowledgment emails
@vortle86 's testing shows that feedback received acknowledgments are going to an empty email address. On staging it's rerouted to the default address, but it's not actually working at all. Probably the token is wrong.

### PR DESCRIPTION
**Pull request by rfay**
Also improves the wording just a bit. Note that I was not able to update this
features or detect that they weren't working using normal tools, so this
is done with a recreate.

### CODE REVIEW

File: docroot/sites/all/modules/dev/ws_rules_features/modules/positive_feedback_rules/positive_feedback_rules.rules_defaults.inc [hunk 1]
```diff
@@ -22,16 +22,16 @@ function positive_feedback_rules_default_rules_configuration() {
       ],
       "DO" : [
         { "mail" : {
-            "to" : "[node:field_member_i_trust:mail]",
-            "subject" : "[Warmshowers.org] You have received member feedback",
+            "to" : [ "node:field-member-i-trust:mail" ],
```
Comment (kalpaitch) [inline docroot/sites/all/modules/dev/ws_rules_features/modules/positive_feedback_rules/positive_feedback_rules.rules_defaults.inc]: Surely that can't be right, the token is [node:field-member-i-trust:mail], not with quote marks in it??? I think maybe it was just the hyphens that needed to change.

**Comment by kalpaitch** [comment]
++ if it works, maybe just check that to token??

**Comment by rfay** [comment]
I was fascinated that the new feature save changed the formatting of the "to" - of course I did use the token selector to get to it, but still...

File: docroot/sites/all/modules/dev/ws_rules_features/modules/positive_feedback_rules/positive_feedback_rules.rules_defaults.inc (hunk 1 above)
Comment (rfay) [inline docroot/sites/all/modules/dev/ws_rules_features/modules/positive_feedback_rules/positive_feedback_rules.rules_defaults.inc]: You're right - good eye, but yes, that's what features spits out! I'll be suspicious also. 

And a weirdness of this one is that changing the rule did not invalidate the feature or anything.

**Pull request merged**